getCustomerId() method in Helper/Data seems wrong

Hi there

There seems to be some problems in the `Helper/Data.php` file, I think the method `getCustomerId()` might not work as expected.
Running [phpstan](https://github.com/phpstan/phpstan/) on level 1 found the following 3 problems:

```
------ -------------------------------------------------------------------------------------------
  Line   module-core/Helper/Data.php
 ------ -------------------------------------------------------------------------------------------
  49     Access to an undefined property Commerce365\Core\Helper\Data::$customerSession.
  53     Access to an undefined property Commerce365\Core\Helper\Data::$httpContext.
  53     Access to constant CUSTOMER_ID on an unknown class Commerce365\Core\Helper\ContextPlugin.
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols
 ------ -------------------------------------------------------------------------------------------
```

I have the impression this method `getCustomerId()` is obsolete, not used, and can be removed entirely?

@TysvdHeuvel, here you go: https://github.com/NVision-Commerce-Solutions/module-core/pull/8

I've added some more fixes, based on new output from [phpstan](https://github.com/phpstan/phpstan) (please consider using this tool going forward, it will force you to keep your code clean - maybe best in combination with https://github.com/bitExpert/phpstan-magento):

```
$ /path/to/phpstan analyse --level=0 .
 33/33 [▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓] 100%

 ------ -------------------------------------------------------------------------------------------
  Line   Helper/Data.php
 ------ -------------------------------------------------------------------------------------------
  49     Access to an undefined property Commerce365\Core\Helper\Data::$customerSession.
         💡 Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property
  53     Access to an undefined property Commerce365\Core\Helper\Data::$httpContext.
         💡 Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property
  53     Access to constant CUSTOMER_ID on an unknown class Commerce365\Core\Helper\ContextPlugin.
         💡 Learn more at https://phpstan.org/user-guide/discovering-symbols
 ------ -------------------------------------------------------------------------------------------

 ------ -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Line   Setup/Patch/Data/AddCustomerCompanyAttributes.php
 ------ -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  42     Method Commerce365\Core\Setup\Patch\Data\AddCustomerCompanyAttributes::apply() should return $this(Commerce365\Core\Setup\Patch\Data\AddCustomerCompanyAttributes) but return statement is missing.
 ------ -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------

 ------ -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Line   Setup/Patch/Data/ChangeParentCustomerAttributeLabel.php
 ------ -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  39     Method Commerce365\Core\Setup\Patch\Data\ChangeParentCustomerAttributeLabel::apply() should return $this(Commerce365\Core\Setup\Patch\Data\ChangeParentCustomerAttributeLabel) but return statement is missing.
 ------ -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------


 [ERROR] Found 5 errors

```

--- a/Helper/Data.php
+++ b/Helper/Data.php
@@ -43,15 +43,4 @@ class Data extends AbstractHelper
     {
         $this->_logger->debug($message);
     }
-
-    public function getCustomerId()
-    {
-        if ($customerId = $this->customerSession->getCustomerId()) {
-            return $customerId;
-        }
-
-        if ($customerId = $this->httpContext->getValue(ContextPlugin::CUSTOMER_ID)) {
-            return $customerId;
-        }
-    }
 }

--- a/Setup/Patch/Data/AddCustomerCompanyAttributes.php
+++ b/Setup/Patch/Data/AddCustomerCompanyAttributes.php
@@ -40,6 +40,8 @@ class AddCustomerCompanyAttributes implements DataPatchInterface
     {
         $this->createAttribute('bc_contact_no', 'BC Contact Number');
         $this->createAttribute('parent_customer_id', 'Parent Customer ID for Contacts');
+
+        return $this;
     }
 
     private function createAttribute($code, $label)

--- a/Setup/Patch/Data/ChangeParentCustomerAttributeLabel.php
+++ b/Setup/Patch/Data/ChangeParentCustomerAttributeLabel.php
@@ -28,14 +28,12 @@ class ChangeParentCustomerAttributeLabel implements DataPatchInterface
 
     public function apply()
     {
-        $this->moduleDataSetup->startSetup();
-
         $this->moduleDataSetup->getConnection()->update(
             $this->moduleDataSetup->getTable('eav_attribute'),
             ['frontend_label' => 'Parent Customer'],
             ['attribute_code = ?' => 'parent_customer_id']
         );
 
-        $this->moduleDataSetup->endSetup();
+        return $this;
     }
 }

